test/unit/general/contextBrokerOAuthSecurityAccess-test.js

```diff
@@ -19,6 +19,8 @@
  *
```

Comment:
Are NGSIv2 unit test planed for this PR? Or they will be implemetented in a next one (also part of #663)?

Reply:
Yes, I plan to do a new PR as part of #663 but I preferred not to do this in a single request just for simplicity. If you agree, when we have the LGTM to this PR, I will do the new one with the NGSIv2 support.

Reply:
Ok. Noted in #663.

NTC

Reply:
Finally, I decided to extend the tests for NGSIv2 in this PR.

Reply:
Fixed with a71f9fd745198cc4050d16984497db3fc6dc80f7